spacy/cli/package.py

```diff
@@ -212,9 +213,18 @@ def get_third_party_dependencies(
         if "factory" in component:
             funcs["factories"].add(component["factory"])
     modules = set()
+    lang = config["nlp"]["lang"]
     for reg_name, func_names in funcs.items():
         for func_name in func_names:
-            func_info = util.registry.find(reg_name, func_name)
+            # Try the lang-specific version and fall back
+            try:
+                func_info = util.registry.find(reg_name, lang + "." + func_name)
+            except RegistryError:
+                try:
+                    func_info = util.registry.find(reg_name, func_name)
+                except RegistryError as regerr:
+                    # lang-specific version being absent is not actually an issue
+                    raise regerr from None
```

Comment:
Is there any need for this `try`/`except` vs. the previous behavior?

Reply:
Without this second `try`/`except`, if the fallback fails the traceback here will always report the first lookup failure, which isn't helpful (that's what I wanted to communicate with the comment, anyway). Using the `from None` makes the traceback shorter and to the point so I figured that was preferable.